### DISCUSSION TITLE
Specify element to focus when glightbox is closed?

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [biati-digital/glightbox#371] [General]
Hi

Is it possible to specify an element that gets focus when the light-box is closed?

I'm doing some accessibility testing, and I've noticed when using the keyboard I can tab to the light-box trigger, press enter to open the light-box. But when I press esc or close the light-box by clicking `gclose gbtn` the focus defaults to `body`

This means keyboard users have to navigate back to the trigger button and beyond to continue navigating the website.

Is there a way to configure `glightbox` so when the light-box is closed `gclose gbtn` receives focus?

Hope that makes sense.

**Comment by stephenmeehanuk** [comment]
I've had another look at the docs, I'm trying to do something like this, but it's not working...

```js
import GLightbox from "glightbox";

var lightbox = GLightbox();
    lightbox.on('close', () => {
        console.log(`Modal close`);
        function setFocus() {
            document.getElementById("glight_box").focus();
        }
});
```

I've added `id="glight_box"` to the light-box trigger.

The `console.log` is working when the light-box is closed, but `id="glight_box"` isn't getting focus?

**Reply by gingerchew** [reply]
I think you're forgetting to run your function:

```js

import GLightbox from "glightbox";

var lightbox = GLightbox();
    lightbox.on('close', () => {
        console.log(`Modal close`);
        function setFocus() {
            document.getElementById("glight_box").focus();
        }
        // here :)
        setFocus()
});

```

But i appreciate you bringing the issue of focus not going back to the initial button/activator to our attention!
